sam_stonefish_sim/scripts/bringup.sh

```diff
@@ -195,6 +198,15 @@ do
 	tmux send-keys "mon launch sam_stonefish_sim mission.launch robot_name:=$ROBOT_NAME bridge_port:=$IMC_BRIDGE_PORT neptus_addr:=$NEPTUS_IP bridge_addr:=$SAM_IP imc_system_name:=$ROBOT_NAME imc_src:=$IMC_SRC max_depth:=$MAX_DEPTH min_altitude:=$MIN_ALTITUDE --name=${ROBOT_NAME}_$(tmux display-message -p 'p#I_#W') --no-start" C-m
 
 	# ADD NEW LAUNCHES THAT ARE SPECIFIC TO ONE SAM HERE
+	tmux select-window -t $ROBOT_SESSION:5
+	tmux send-keys "mon launch tf_convenience_topics odom_listener.launch robot_name:=$ROBOT_NAME --name=${ROBOT_NAME}_$(tmux display-message -p 'p#I_#W') --no-start" C-m
```

Comment:
This should probably be part of the `robot_bridge.launch` launch file rather than have its own tmux window.

Reply:
And it should be part of the lolo simulation as well. I can add that once added to sam.

Reply:
The `robot_bridge.launch` should only start the `tf_convenience_topics` if the `simulate_dr` argument is set to `true`. Otherwise it should be launched by the DR launch files.

Reply:
That can be done with an `if` clause in the `include` tag.

Reply:
You can see how it can be done on this line: https://github.com/smarc-project/smarc_stonefish_sims/blob/noetic-devel/sam_stonefish_sim/launch/robot_bridge.launch#L14 . Note that if you uncomment that, you can use `/sam/dr/odom` instead of `/sam/sim/odom`.

Reply:
That makes sense. I thought we would launch it as part of DR, that's why I had the separate window. I am updating it now! About republishing to /sam/dr/odom, that would actually make things much easier with the topics, since then we have the same topics on the simulated and real versions. Shall we keep that as default?

Reply:
I've updated robot_bridge.launch with the launch file now, and removed it from the bringup script.

Reply:
I've also added tf_convenience_topics as exec_depend.